### PR TITLE
Add force parameter

### PR DESCRIPTION
**Pull request by justnoxx**

### CODE REVIEW

File: pages/EC-WebLogic_help.xml [hunk 1]
```diff
@@ -3095,14 +3145,78 @@
...
 
-            </div>
-		
-		<div id="StopManagedServer">
+        <div id="StopCluster"> <a name="StopCluster" id="StopCluster"></a>
+            <h2>StopCluster</h2>
+            <p>This procedures stops a cluster.</p>
+            <h3>Input</h3>
+            <ol>
+                <li>In the Main Menu, click <b>Admin</b> &gt; <b>Plugins</b> to open the Plugin Manager.</li>
```
Comment (imago-storm) [inline pages/EC-WebLogic_help.xml]: I thought the common use case is to use the procedure from the project, not from the plugin itself.

File: src/main/resources/project/jython/stop_cluster.jython [hunk 2]
```diff
@@ -38,15 +39,46 @@ clusterName = r'''
...
+
+ignoreSessions = r'''
+[% ignoreSessions %]
+'''.strip()
+
+forceShutdown = r'''
+[% forceShutdown %]
+'''.strip()
+
+timeout = int(timeout);
```
Comment (imago-storm) [inline src/main/resources/project/jython/stop_cluster.jython]: Can it be an empty string? In this case this string will fail.

File: src/main/resources/project/jython/stop_cluster.jython [hunk 3]
```diff
@@ -38,15 +39,46 @@ clusterName = r'''
 [% cluster_name %]
 '''.strip()
 
+timeout = r'''
+[% timeout %]
```
Comment (imago-storm) [inline src/main/resources/project/jython/stop_cluster.jython]: Why not shutdownTimeout, like the parameter name? It's not very convenient, to remember all the names.

Comment (imago-storm) [inline src/main/resources/project/jython/stop_cluster.jython]: I know, still I don't get why it has a different name. forceShutdown for example has the same name as a parameter.

Comment (justnoxx) [inline src/main/resources/project/jython/stop_cluster.jython]: 1) .jython templates are designed for internal use.
2) shutdownTimeout parameter is being modified at the perl side of procedure, it is not a regular property forwarding, as forceShutdown or ignoreSessions.

**Pull request merged**